python/pyspark/sql/udtf.py

```diff
@@ -372,22 +373,43 @@ def _create_judtf(self, func: Type) -> "JavaObject":
             )
         return judtf
 
-    def __call__(self, *args: "ColumnOrName", **kwargs: "ColumnOrName") -> "DataFrame":
+    def __call__(
+        self, *args: Union["ColumnOrName", "TableArg"], **kwargs: Union["ColumnOrName", "TableArg"]
```

Comment:
The type hint should be `TableValuedFunctionArgument`?

Reply:
Changed to TVFArgumentOrName thanks!